### PR TITLE
BZ#1486224 - Refactor sui group switching to use api

### PR DESCRIPTION
**Pull request by AllenBW**
https://bugzilla.redhat.com/show_bug.cgi?id=1486224

Right now it uses headers, taks a LOOOONNNGG time to take effect, most of the time feels broken, gonna switch this over to using api.

In order to keep excessive api calls down, require this work to be completed first
- [x] https://github.com/ManageIQ/manageiq-api/issues/74

### So what did we do? 
groupswitch

mad props out to @jntullo for helpin' this reality come to 💡

### CODE REVIEW

**Comment by chriskacerguis** [comment]
@AllenBW this ready to go?

**Comment by AllenBW** [comment]
@chriskacerguis Nope, still gotta do work, above mentioned pr was just merged in

File: client/app/core/session.service.js [hunk 1]
```diff
@@ -30,24 +29,33 @@ export function SessionFactory ($http, $q, $sessionStorage, $window, $state, $co
     $sessionStorage.token = model.token
   }
 
-  function setMiqGroup (group) {
+  function setGroup (group) {
     $http.defaults.headers.common['Accept'] = 'application/json;charset=UTF-8'
-    $http.defaults.headers.common['X-Miq-Group'] = unescape(encodeURIComponent(group))
-    $sessionStorage.miqGroup = group || null
-    $sessionStorage.selectedMiqGroup = group
+    if (typeof group === 'object') {
```
Comment (AllenBW) [inline client/app/core/session.service.js]: i know this is ugly,
> why not just do `foo = group.stuff || group` ???

I WANT TO but our tests that rely on this work fail when this is written shorthand, i think it has something to do with running in strict mode 🤔

File: client/app/layouts/_navigation.sass [hunk 2]
```diff
@@ -45,7 +45,7 @@
     padding-top: 19px
 
   .dropdown-menu
-    margin-top: 7px;
+    margin-top: 7px
```
Comment (AllenBW) [inline client/app/layouts/_navigation.sass]: exceeds the scope of this pr, buttttt it was broken :-/

**Comment by AllenBW** [comment]
screen shot 2017-09-28 at 3 14 35 pm

Updated da message

**Comment by JPrause** [comment]
@chriskacerguis can you set the Milestone to Sprint 70

**Comment by chriskacerguis** [comment]
@JPrause done.

**Pull request merged**